### PR TITLE
UI fixes and niceties.

### PR DESCRIPTION
**Pull request by mstratford**
Just a growing collection of stuffs that would be nicer if...

- Say welcome instead of "Menu".
- Reset the selected timeslot when impersonating if the user doesn't have permission to access it
- Explain what an audio log is
- Remove non-manual uploads

### CODE REVIEW

**Comment by markspolakovs** [comment]
@mstratford should this still be open?

**Comment by mstratford** [comment]
Yeah, still look like valid changes to me :)

File: src/Templates/NIPSWeb/manage_library.twig [hunk 1]
```diff
@@ -1,30 +0,0 @@
-<!DOCTYPE html>
```
Comment (markspolakovs) [inline src/Templates/NIPSWeb/manage_library.twig:1]: Should the associated controller be deleted as well?

**Review by markspolakovs — APPROVED** [review]

File: src/Templates/NIPSWeb/manage_library.twig (hunk 1 above)
Comment (mstratford) [reply src/Templates/NIPSWeb/manage_library.twig:1]: I forget, if it's not the same as the manual upload training one, then yes.

Comment (mstratford) [reply src/Templates/NIPSWeb/manage_library.twig:1]: Doesn't look like it's actually in the code anymore, just the manual upload one is in there.

**Pull request merged**